CHANGELOG.md

```diff
@@ -23,9 +23,11 @@
 - don't show logo twice in notifications (because macOS already shows applogo)
 - disable editing of left/readonly groups
 - fix member list being incomplete in chat requests
+- fix setting selfstatus
```

Comment:
do we need to include bugfixes for bugs that were introduced in this version in the changelog don't they cancel each other out?

Reply:
Ah right, that should go away again